tiddlers/Tidme.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "created": "20141029161248510",
+        "text": "[img width=100 [$:/favicon.ico]]\n\n! Wissen, das ein Leben lang hält, tief in deinem Geist \nn Mit dem fortschrittlichen Konzept des [[Progressive Learning|https://www.yuque.com/supermemo/wiki/incremental_learning]] ist [[Ink Drawer]] das zweite Gehirn, das dir hilft, dein ganzes Leben lang Wissen zu erwerben.\n",
```

Comment:
It might be better to translate it like this:`[[Progressive Learning|https://help.supermemo.org/wiki/Incremental_learning]] ist [[Tidme]]`

Reply:
Oops, that was an oversight